Unix/doc/setup-kerberos-omi.md

```diff
@@ -0,0 +1,627 @@
+IPV6_ADDR_GEN_MODE="stable-privacy"
+NAME="eth0"
+UUID="xxxxxxxxxx"
+DEVICE="eth0"
+ONBOOT="yes"
+PEERDNS="yes"
+PEERROUTES="yes"
+IPV6_PEERDNS="yes"
+IPV6_PEERROUTES="yes"
+IPV6_PRIVACY="no"
+DNS1=10.xxx.xxx.xxx
+DNS2=10.xxx.xxx.xxx
+DNS_DOMAIN=contoso.com
+DNS_SEARCH=contoso.com
+```
+
+ If Network manager is active, it may be necessary to remove the name server entries already in the resolv.conf.
+ If Network Manager is not active, just edit /etc/resolv.conf. Either way, /etc/resolv.conf should end up looking like 
+ (where xx are valid ip addresses:
+
+```
+domain contoso.com
+search contoso.com
+nameserver 10.xx.xx.xx
+nameserver 10.xx.xx.xx
+
+```
+If you are using network manager and add the DNSx and DNS_DOMAIN entries, and they dont show up in resolv.conf, chances
+are that the ONBOOT entry is set to off. Set it to on and try again.
+
+To verify, you should be able to ping a known host on the network, and ping the fully qualified host name from another
+host. It will probably be necessary to add the dns record for the new host on the ADC using the DNS tool, in the
+"DNS forward zones" category.
+
+When you are done, the critical thing is the default hostname is a fully qualified domain name, so
+```
+   # hostname
+   myhost.contoso.com
+```
+
+   This is critical. Make sure to resolve this before proceding. If it is not right, look in /etc/sysconfig/network at the
+   HOSTNAME entry. 
+
+ 
+#### authconfig
+
+ Redhat (and centos, and fedora) have a utility, *authconfig*, which does the majority of the configuration automatically.  
+ The command line is:
+```
+         authconfig --enablesssd --enablesssdauth --enablelocauthorize --enablekrb5 --update --krb5realm=CONTOSO.COM --krb5kdc=mykddc.contoso.com
+```
+
+#### Verify/setup the kerberos environment
+
+At the end of this, you should be able to 
+```
+    kinit domain_user@CONTOSO.COM
+```
+where domain user is a known user already defined in the ADC.
+Then list the credential using
+```
+   klist domain_user@CONTOSO.COM
+```
+
+
+#### Join the domain using realm
+
+If you have a correct setup, you shold be able to see the specifics using realm.
+
+
+```
+     sudo realm info CONTOSO.COM
+```
+
+Note that real will not join if there is an /etc/sssd/sssd.conf.  If you have one, move it to /etc/sssd/sssd.conf.old.
+Then:
+
+```
+     sudo realm join CONTOSO.COM -U Administrator
+```
+
+You can verify this initially by doing a getent:
+```
+    getent passwd domain_user@CONTOSO.COM
+```
+If that worked, you can actually log into your domain user:
+```
+    % su - root
+    password for root:
+    # su - domain_user@CONTOSO.COM
+    domainuser@CONTOSO.COM % exit
+    # exit
+    %
+```
+
+and exit.  
+
+#### Add the HTTP service principal
+
+To use OMI, you need to talk to the HTTP service principal for the host. This principal is not created by default
+and needs to be added. If you have access, you could log into the ADC and use *setspn*. This is *not* the recommended route.
+
```

Comment:
Wait, I'm confused. You say this is *not* the recommend route, and then you go on to describe it in detail? Is this an error?